caravel/assets/javascripts/components/VictoryTheme.js

```diff
@@ -0,0 +1,175 @@
+const { assign } = Object;
```

Comment:
unrelated question, if we are not using `babel-polyfill` are we relying on `Object.assign` coming from chrome / the browser?